com.neoutils.NeoRegex.yml

```diff
@@ -0,0 +1,37 @@
+app-id: com.neoutils.NeoRegex
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: NeoRegex
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+
+environment:
```

Comment:
which page in the document say this is valid?

(so I can get it fixed)

Reply:
It wasn't based on the documentation. I confess that I didn't even consider the possibility of it being invalid. I apologize, I will correct it.

Reply:
It wasn't even necessary anymore. It ended up remaining as leftover from the initial tests.